static/webpack.config.js

```diff
@@ -126,6 +126,10 @@ const config = {
       __dirname + "/js/apps/homepage/main.ts",
       __dirname + "/css/homepage.scss",
     ],
+    homepage_custom_dc: [
+      __dirname + "/js/apps/homepage/main_custom_dc.ts",
+      __dirname + "/css/homepage.scss",
+    ],
```

Comment:
I see both `homepage` and `homepage_custom_dc` entry points use `/css/homepage.scss`. Is the intention for both main and custom homepages to share `/css/homepage.scss`? Why not give each its own scss?

Reply:
custom dc has a way to override which is in "static/custom_dc/custom/overrides.css". Here needs to use either base.css or hompage.css to make the default UI works.